Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN chown -R appuser:appgroup /usr/src/app/
 USER appuser
 USER 1000
 
-RUN RAILS_ENV=production bundle exec rake assets:clean assets:precompile assets:non_digested SECRET_KEY_BASE=required_but_does_not_matter_for_assets 2> /dev/null
+RUN RAILS_ENV=production AWS_ACCESS_KEY_ID=not_real AWS_SECRET_ACCESS_KEY=not_real bundle exec rake assets:clean assets:precompile assets:non_digested SECRET_KEY_BASE=required_but_does_not_matter_for_assets 2> /dev/null
```

Comment:
@Jav  (he/him),  just bring you attention to this one line change,  the reason for this is that  somehow I believe the aws sdk changed the way of checking credential when initialising a bucket object in aws.rb.  The above rake tasks do not need to connect s3 but it will run aws.rb as part fo config/intializers, so I passed the fake key similar as the `SECRET_KEY_BASE=required_but_does_not_matter_for_assets` to make this step work.